website/docs/networking/vpc-lattice/gatewayapicontroller.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "AWS Gateway API Controller"
+sidebar_position: 10
+---
+
+Gateway API is an open-source project managed by the Kubernetes networking community. It is a collection of resources that model application networking in Kubernetes. Gateway API supports resources such as GatewayClass, Gateway, and Route that have been implemented by many vendors and have broad industry support.
+
+Originally conceived as a successor to the well-known Ingress API, the benefits of the Gateway API include (but are not limited to) explicit support for many commonly used networking protocols, as well as tightly integrated support for Transport Layer Security (TLS).
+
+At AWS, we implement the Gateway API to integrate with Amazon VPC Lattice. AWS Gateway API Controller is an open-source project and fully supported by Amazon. When the AWS Gateway API controller is installed in your cluster, the controller watches for the creation of Gateway API resources such as gateways and routes and provisions corresponding Amazon VPC Lattice objects as shown above.
```

Comment:
"At AWS, we implement the Gateway API to integrate with Amazon VPC Lattice." Without some elaboration, for a new reader I sense, a "chicken-and-egg" situation here. "integrate with VPC Lattice" implies existence of "something" but we haven't explained what exists. Would it explain things better if we add some additional details about this integration?